movement/sample_data.py

```diff
@@ -32,6 +34,28 @@
 METADATA_FILE = "metadata.yaml"
 
 
+@contextmanager
+def hide_pooch_hash_logs():
+    """Hide only SHA256 hash printouts from pooch.retrieve when known_hash=None."""
```

Comment:
This line is longer than our limit. I would make the descrption shorter, which also makes our linter happy.

```suggestion
    """Hide SHA256 hash printouts from ``pooch.retrieve``."""
```